sys-auth/libyubikey/libyubikey-1.13.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="Low level C SDK for the Yubico YubiKey authentication device"
+DESCRIPTION="This package make up the low-level C software development kit for the Yubico YubiKey authentication device."
+HOMEPAGE="https://developers.yubico.com/yubico-c/"
+SOURCE_URI="https://developers.yubico.com/yubico-c/Releases/libyubikey-1.13.tar.gz"
+CHECKSUM_SHA256="04edd0eb09cb665a05d808c58e1985f25bb7c5254d2849f36a0658ffc51c3401"
+REVISION="1"
+LICENSE="BSD (2-clause)"
+COPYRIGHT="Yubico AB"
```

Comment:
Could you please change the order of these lines to:
```
SUMMARY="Low level C SDK for the Yubico YubiKey authentication device"
DESCRIPTION="This package make up the low-level C software development kit for the \
Yubico YubiKey authentication device."
HOMEPAGE="https://developers.yubico.com/yubico-c/"
COPYRIGHT="2008-2015 Yubico AB"
LICENSE="BSD (2-clause)"
REVISION="1"
SOURCE_URI="https://developers.yubico.com/yubico-c/Releases/libyubikey-$portVersion.tar.gz"
CHECKSUM_SHA256="04edd0eb09cb665a05d808c58e1985f25bb7c5254d2849f36a0658ffc51c3401"
```
(Minor changes to COPYRIGHT and re-wrapped DESCRIPTION.)

(See the [HaikuPorter Guidelines](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering) for more info)